doc/release/1.4.0-notes.rst

```diff
@@ -24,12 +24,49 @@ For running on PyPy, PyPy3 6.0+ and NumPy 1.15.0 are required.
 Highlights of this release
 --------------------------
 
--
+- a new submodule, `scipy.fft`, now supersedes `scipy.fftpack`; this
```

Comment:
Should these be single or double backticks?

Reply:
Single is good here. They will turn into names. So you can do that only for fully qualified names, e.g. `scipy.xxx.yyy`